packages/flutter/test/cupertino/segmented_control_test.dart

```diff
@@ -1314,7 +1314,7 @@ void main() {
       find.byType(RepaintBoundary),
       matchesGoldenFile('segmented_control_test.0.0.png'),
     );
-  }, skip: !Platform.isMacOS);
+  }, skip: !Platform.isLinux);
```

Comment:
There should be another place where this needs to be changed

Reply:
Line 1276

Reply:
Good catch! Will update shortly.